astropy/modeling/utils.py

```diff
@@ -493,3 +493,190 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def ellipse_extent(a, b, theta):
+    """
+    Calculates the extent of a box encapsulating a rotated 2D ellipse.
+
+    Parameters
+    ----------
+    a : float
+        Major axis
+    b : float
+        Minor axis
+    theta: float
+        Rotation angle in radians.
+
+    Returns
+    -------
+    image : array
+            The evaluated model on the input image or coordinates. If 
+            no image or coordinates are input, the returned array is 
+            limited to the bounding_box of the model. The the bounding_box
+            is not defined, an image or coordinates must be passed.
+
+    Examples
+    --------
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        import matplotlib.pyplot as plt
+        from astropy.modeling.models import Ellipse2D
+        from astropy.modeling.utils import ellipse_extent
+
+        amplitude = 1
+        x0 = 50
+        y0 = 50
+        a = 30
+        b = 10
+        theta = np.pi/4
+
+        model = Ellipse2D(amplitude, x0, y0, a, b, theta)
+
+
+        dx, dy = np.around( ellipse_extent(a, b, theta) )
+
+        limits = [slice(y0 - dy, y0 + dy + 1, 1),
+                  slice(x0 - dx, x0 + dx + 1, 1)]
+
+        coords = np.mgrid[limits][::-1]
+
+        plt.imshow(model(*coords),cmap='binary',interpolation='nearest',alpha=.5,
+                  extent = [x0 - dx, x0 + dx, y0 - dy, y0 + dy])
+        plt.show()
+    """
+
+    t = np.arctan2(-b * np.tan(theta), a)
+    dx = a * np.cos(t) * np.cos(theta) - b * np.sin(t) * np.sin(theta)
+
+    t = np.arctan2(b, a * np.tan(theta))
+    dy = b * np.sin(t) * np.cos(theta) + a * np.cos(t) * np.sin(theta)
+    return np.abs([dx, dy])
+
+
+def render_model(model, image=None, coords=None, copy=True):
+    """
+    Evaluates a model on an input array. Evaluation is limited to
+    a bounding box if the `bounding_box` property is defined for
+    the model.
+
+    Parameters
+    ----------
+    model : Model
+            Model to be evaluated
+    image : array
+            Array on which the model is evaluated.
+    coords: list
```

Comment:
Again, add space `coords :`
Sphinx is very picky, I think (but am not sure) this will not look OK in HTML output.
Did you check your docstrings in HTML already?